ddtrace/internal/constants.py

```diff
@@ -103,3 +103,9 @@ class _PRIORITY_CATEGORY:
 }
 _KEEP_PRIORITY_INDEX = 0
 _REJECT_PRIORITY_INDEX = 1
+
+
+# List of support values in DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED
+class EXPERIMENTAL_FEATURES:
+    # Enables submitting runtime metrics as gauges (instead of distributions)
+    RUNTIME_METRICS = "DD_RUNTIME_METRICS_ENABLED"
```

Comment:
Does this mean the old usage of DD_RUNTIME_METRICS_ENABLED should be deprecated in favor of the experimental support?

Reply:
Yup and we plan to do this in a minor release 😅